tests/drivers/spi/spi_loopback/boards/frdm_mcxn236.overlay

```diff
@@ -5,7 +5,7 @@
  */
 
 /* To test this sample, connect
- * LPSPI3 MOSI(J2-8, P1_0/FC3_P0)     -->        LPSPI5 MISO(J2-10, P1_2/FC3_P2)
+ * LPSPI3 MOSI(J2-8, P1_0/FC3_P0)     -->        LPSPI3 MISO(J2-10, P1_2/FC3_P2)
```

Comment:
personally, I think you can squash this change into the board change commit, since it is so trivial, doesnt really seem to need its own commit, but thats just my opinion, not blocking